fetch/main.js

```diff
@@ -0,0 +1,21 @@
+async function getArray(url, options) {
+  try {
+    const response = await fetch(url);
+    if (!response.ok) {
+      throw new Error('Network response failed');
```

Comment:
Actually, the network was fine, which is why you got a response. It's just that the server returned a status that was not in the 200 range.